iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesParserTest.java

```diff
@@ -512,4 +513,31 @@ void shouldRethrowParseExceptionsWithDifferentDetails(@Nullable String details)
         .isInstanceOf(ParseException.class);
     }
   }
+
+  @Test
+  void shouldParseValuesYamlFileAsSimpleKubernetesFile() {
+    when(inputFile.toString()).thenReturn("chart/values.yaml");
+    when(inputFile.filename()).thenReturn("values.yaml");
+
+    var actual = parser.parse("foo: bar", inputFileContext);
+
+    assertThat(actual.template()).isEqualTo(FileTree.Template.NONE);
```

Comment:
~I don't follow; shouldn't `KubernetesParser#validateInputFileContext` return an empty tree marked as Helm template for this case?~
OK, it's clear after I looked at the second test. Then I suggest to rename this one to `shouldParseValuesYamlFileWithoutHelmContentAsSimpleKubernetesFile`